### DISCUSSION TITLE
3D ODF plot

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#1586] [Ask Anything]
Hi， everyone，
   I am wondering if there is a way to plot 3D odf as below.
   Thanks!

   Brian
![image](https://user-images.githubusercontent.com/69452222/194759631-de3ced1b-efc8-4d48-a285-47ba6d7195d4.png)

**Comment by kilir** [comment]
Hi,

see ODF/plot3d.

Cheers,
Rüdiger
